volume/store/store_windows.go

```diff
@@ -0,0 +1,12 @@
+package store
+
+import "strings"
+
+// normaliseVolumeName is a platform specific function to normalise the name
+// of a volume. On Windows, as NTFS is case insensitive, under
+// c:\ProgramData\Docker\Volumes\, the folders John and john would be synonymous.
+// Hence we can't allow the volume "John" and "john" to be created as seperate
+// volumes.
+func normaliseVolumeName(name string) string {
```

Comment:
Wouldn't this be more appropriate in the local driver rather than at the volume store level?